src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -3808,6 +3815,33 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         End Sub
 
+        Private Sub CheckTupleNamesAgainstBase(type As NamedTypeSymbol, diagnostics As DiagnosticBag)
```

Comment:
>CheckTupleNamesAgainstBase [](start = 20, length = 26)

Should this method be Shared? Also, I believe similar method lives in ConstraintsHelper.cs on C# side.